stories/card/card.stories.js

```diff
@@ -53,6 +54,61 @@ export const CardAnatomy = () => `<div style="display:flex; justify-content:spac
                 </div>
             </a>
             <div class="fd-card__content" role="group" aria-label="Card Content"></div>
+            <div class="fd-card__footer">
+                <div class="fd-card__footer__actions">
```

Comment:
I don't like too much of block selectors. Since we don't have actions in card, maybe we can simplify the selectors?
Something like: `fd-card__footer__actions` -> `fd-card__actions` and `fd-card__footer__actions__item` -> `fd-card__action-item`?

Reply:
Yes we could and I like it that way too, but this way in CSS side everything is inside __footer without using any .#{$block} or something like it. Since we only want them inside footer and outside of it to not have any effect

Reply:
I'd still prefer more simpler naming due to readability reasons. Anyway, I'd like to ask @InnaAtanasova her opinion too